batch/indexer_Position_Bond.py

```diff
@@ -16,6 +16,18 @@
 
 SPDX-License-Identifier: Apache-2.0
 """
+import log
+from app.utils.web3_utils import Web3Wrapper
+from app.contracts import Contract
+from app.model.db import (
+    Listing,
+    IDXPosition
+)
+from app.config import (
+    DATABASE_URL,
+    TOKEN_LIST_CONTRACT_ADDRESS,
+    ZERO_ADDRESS,
+)
```

Comment:
This module is ran as follows.
> python batch/indexer_Position_Bond.py
so, doesn't the `app` module fail to load?

Reply:
```
[-bash ibet-Wallet-API]$ python batch/indexer_Position_Bond.py
Traceback (most recent call last):
  File "batch/indexer_Position_Bond.py", line 20, in <module>
    from app.utils.web3_utils import Web3Wrapper
ModuleNotFoundError: No module named 'app'
```

Reply:
I don't know why but It looks work on Unit Testing...🤔
I'll try to check, thanks.

Reply:
resolved because error handling is unnecessary 😇